internal/server/audit/logfile/logfile.go

```diff
@@ -0,0 +1,63 @@
+package logfile
+
+import (
+	"encoding/json"
+	"os"
+	"sync"
+
+	"github.com/hashicorp/go-multierror"
+	"go.flipt.io/flipt/internal/server/audit"
+	"go.uber.org/zap"
+)
+
+const sinkType = "logfile"
+
+// LogFileSink is the structure in charge of sending Audits to a specified file location.
+type LogFileSink struct {
```

Comment:
```suggestion
type Sink struct {
```
since the package is `logfile` so this will be exported as `logfile.Sink`